modules/custom-cert-new-install-operator.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * configuration/add-custom-certificates.adoc
+:_module-type: CONCEPT
+[id="custom-cert-new-install-operator_{context}"]
+= Adding a custom certificate during a new installation with Operator
+
+By default, Central only serves an internal TLS certificate, which means you need to handle TLS termination at the ingress or load balancer level.
+If you want to terminate TLS in Central and serve a custom server certificate, specify a secret containing the certificate and private key using the `central.defaultTLSSecret` configuration option.
```

Comment:
This doesn't have to do with termination. It has to do with the certificate central serves. 

This should be "If you want to use a customer certificate...."

Reply:
Also we should just walk them through how to set this directive in the UI of the Web Console.